be/src/vec/jsonb/serialize.cpp

```diff
@@ -107,7 +107,7 @@ void JsonbSerializeUtil::jsonb_to_block(const DataTypeSerDeSPtrs& serdes, const
             MutableColumnPtr col = column_type_name.column->assume_mutable();
             if (col->size() < num_rows + 1) {
                 DCHECK(col->size() == num_rows);
-                if (default_values[i].empty()) {
+                if (default_values[i].empty() || col->empty()) {
```

Comment:
maybe it's wrong, the col always empty for the start, so when `default_values[i]` is not emtpy, it may result incorrect value